src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts

```diff
@@ -84,4 +84,25 @@ export class InputActionDialog {
   close() {
     this.modal.close();
   }
+
+  checkIfJenkinsIsUp() {
+    let url = 'https://jenkins.openshift.io';
+    this.http.get(url)
+      .subscribe((response) => {
+        if (response.status == 200) {
+          console.log('Got ' + response.status + ', Jenkins is up and running...');
+          this.invokeUrl(this.inputAction.proceedUrl);
+          return;
+        } else if (response.status == 307 || response.status == 302) {
+          console.log('Got ' + response.status + ', Jenkins Proxy is logging you in...');
+        } else if (response.status == 503) {
+          console.log('Got ' + response.status + ', Cluster resources capacity is full. waiting along..');
+        } else if (response.status == 202) {
+          console.log('Got 202, Jenkins is currently idled. Please wait while we start it. waiting along..');
+        } else {
+          console.error ('Got status ' + response.status);
+        }
+        setTimeout (this.checkIfJenkinsIsUp(), 10000);
```

Comment:
Looks like infinite loop isn't it? It just finishes if we get a `200`. Shouldn't we finish this loop if we get an unexpected error? And also add some feedback to user? Something like spinning animation in case of unidling?